tests/model/test_Model.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+from PKPD.model import model
+import matplotlib.pyplot as plt
+import myokit
+import numpy as np
+import os
+
+
+class TestModel(unittest.TestCase):
+
+    file_name = 'PKPD/mmt/one_compartment.mmt'
+    models = model.Model()
+
+    def setUp(self) -> None:
+        self.models.mmtfile = 'PKPD/mmt/one_compartment.mmt'
+        self.models.read_mmt_file()
+
+    def test_read_mmt_file(self):
+        self.assertTrue(self.models.initial_values)
+        self.assertTrue(self.models.central_compartment_name)
+        self.assertEqual(self.models.dimension, len(self.models.initial_values))
+        self.assertTrue(type(self.models.params) is dict)
+
+    def test_set_get_mmt_file(self):
+        self.models.set_mmt_file('test')
+        self.assertTrue(self.models.mmtfile == 'test')
+        self.assertEqual(self.models.get_mmt_file(), 'test')
+
+    def test_set_get_initial_values(self):
+        vals = self.models.get_initial_values()
+        new_vals = list(range(0,len(vals)))
+        self.models.set_initial_values(new_vals)
+        self.assertEqual(self.models.initial_values, new_vals)
+        self.assertEqual(self.models.model.state(), new_vals)
+
+    def test_set_get_model(self):
+        self.assertTrue(type(self.models.model) == myokit._model_api.Model)
+        self.assertEqual([n.qname() for n in self.models._get_model().variables()], [n.qname() for n in myokit.load(self.models.mmtfile)[0].variables()])
+
+    def test_set_get_protocol(self):
+        #self.models.set_protocol('model.mmt')
+        #self.assertTrue(print(self.models.protocol) == '[[protocol]]\n# Level  Start    Length   Period   Multiplier\n1.0      0.0      0.1      0.0      0')
+        pass #not required yet
+
+    def test_set_get_params(self):
+        qname = 'param.CL'
+        value = 4.0
+        self.models.set_params(CL = value)
+        self.assertEqual(self.models.model.get(qname).eval(), value)
+        param_dict = {'param.CL': 4.0, 'param.V_c' : 5.0}
+        self.assertEqual(param_dict, self.models.get_params())
+
+    def test_solve(self):
+        #Checks Max Error Pre Spike < 1e-8
```

Comment:
It might be better not to have plots in our automated tests as nobody will look at them when Travis runs them and I'm not sure what happens when we keep producing plots on the server.